package com.mutualmobile.praxis.domain.exceptions

class NetworkErrorException : Throwable() {
}